newIDE/app/src/GameDashboard/GameRegistration.js

```diff
@@ -69,13 +68,13 @@ export const GameRegistration = ({
 
       const { id } = profile;
       setError(null);
-      setUnavailableReason(null);
       try {
         const game = await getGame(
           getAuthorizationHeader,
           id,
           project.getProjectUuid()
         );
+        setUnavailableReason(null);
```

Comment:
Why this change? (curious, I've not thought about it)

Reply:
I've noticed that the call to game can happen multiple times sometimes (probably a hidden problem?). 
In this case, I didn't have access to the game, so the unavailableReason was 'unauthorized', meaning the value will switch from 'null' to 'unauthorized' making the component blink from 'something to display' to 'loader' multiple times.